dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22ClientWindowFaces40.war/resources/index.xhtml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * Copyright (c) 2015, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ -->
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://xmlns.jcp.org/jsf/html"
+      xmlns:f="http://xmlns.jcp.org/jsf/core">
+    <h:head>
+        <title>JSF 2.2 Test Page for Client Window</title>
```

Comment:
Do the tests look for this specific string? Since this is used for Faces 4.0 would be nice to update the title in these htmls to say Faces 4.0 Test Page but if it's to much of a hassle I'm okay leaving it as is.

Reply:
@isaacrivriv  I've made the changes. Would you mind giving a final review?